### DISCUSSION TITLE
Where can I find the Algorithm of Lithium Plating?

### DISCUSSION THREAD

**Discussion by Nithin3297** [pybamm-team/PyBaMM#3838] [Q&A]
Hello guys, 
Can someone explain or let me know where to find the algorithm used for solving Lithium plating governing equations? I found the equations in the paper of Dr. Simon O'Kane. I want to look at the algorithm used to solve the numerical methods used. (By solver). I want to know what numerical method is used and how it is implemented. 
TIA.

**Comment by valentinsulzer** [comment]
There is no special algorithm for plating, it's the standard numerical solving procedure for all models (finite volume discretization in space then solving the resulting system of ODEs/ DAEs using a third-party numerical solver like SUNDIALS)
